docs/peggy/Peggy-Ceth-Account-Set.md

```diff
@@ -9,7 +9,9 @@ The transaction is priviledged, and only the admin account can set it.
 The admin account is the same account that updates the white list validators. The command is as follows:
 
 1. command format
-sifnodecli tx ethbridge update_ceth_receiver_account [cosmos-sender] [ceth-receiver-account]
+```
+sifnodecli tx ethbridge update_ceth_receiver_account $clp_admin_address $ceth-receiver-account
+```
 
 2. command example
```

Comment:
can we remove this example and replace it with the real-world runbook command we will actually run? as above i think?

Reply:
That's fine. I would say it should look like this:

```
sifnodecli tx ethbridge update_ceth_receiver_account $sender_address $ceth-receiver-account
```

Reply:
@ElliotFriedman to confirm, the first param is the sender - who do you mean by this? the admin? in that case does the following change look correct?

```
sifnodecli tx ethbridge update_ceth_receiver_account $admin_address $ceth-receiver-account
``

Reply:
@musnit Yes, the first param is the sender, which should be the admin. That change looks correct.